tortoise/verifying_tortoise.go

```diff
@@ -362,6 +371,13 @@ func (t *turtle) BaseBlock(ctx context.Context) (types.BlockID, [][]types.BlockI
 				log.Int("support_count", len(exceptionVectorMap[1])),
 				log.Int("neutral_count", len(exceptionVectorMap[2])))
 
+			metrics.LayerDistanceToBaseBlock.
+				With("last_layer", t.Last.String()).
+				With("last_evicted", t.LastEvicted.String()).
+				With("base_block_layer", layerID.String()).
+				With("block_id", blockID.String()).
```

Comment:
are these data last_layer/last_evicted/base_block_layer/block_id really going to be useful?

Reply:
I think that `last_layer` and `block_id` may be useful to find a source of an issue. The rest may be removed